### DISCUSSION TITLE
Deployment option not showing in GitHub Pages

### DISCUSSION THREAD

**Discussion by urxmedia** [community/community#113194] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am trying to host a static website on GitHub Pages but the option for selecting the branch and deploying the site is not visible under my Pages tab. The only things that is visible is Verified Domains. The pages section is completely empty.

I have added my custom domain here but why can't I see the Deploy option.

![image](https://github.com/community/community/assets/94640190/99ab1596-2f60-4fc4-88ea-433b0f6d9d13)

Has anyone faced this issue? Am I doing anything wrong? Is there any setting that needs to be enabled for this?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @urxmedia 👋🏼 , 

It looks like you're in your GitHub account settings, for GitHub Pages most settings will live in the repository that hosts the GitHub Pages site. 
 
![image](https://github.com/community/community/assets/92824518/e312d7c9-da16-4f14-bafa-8ef1ca231228)


Check out [our doc on getting started](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) for more details.

**Reply by urxmedia** [reply]
Thanks. Got it now. I had completely overlooked the Repository settings.

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.

**Comment by tbathat** [comment]
Hey guys! I came across your discussion because I'm dealing with the same issue. However, the stage and deployment tabs don't show up for me in the repository settings.

![image](https://github.com/user-attachments/assets/561f71c9-affd-47f8-aad0-cb6882ad31cd)

**Reply by LiteBrite82** [reply]
It looks like you need to make the repo public to use Pages with your current account type.
